### ISSUE TITLE
Links on About page should open separate tab

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#2541]
**Describe the bug**
The links on the About page go directly to the page in the link, moving off the SEED instance.

**Expected Behavior**
I expected that when I clicked on a link, a new tab would open up for that linked page

**Actual Behavior**
See Description above

**Steps to Reproduce**
1. Go to About page on SEED instance
2. Click on one of the links, such as the More Details link
3. The page will go directly to the linked page, rather than open up a new tab. 

**Instance Information**
Instance: LBNL staging server
SHA: bba6e024a

**Additional Context**
none

**Comment by RDmitchell** [comment]
@adrian-lara -- is this on dev1? I am not seeing the links being opened in a separate tab.

**Comment by adrian-lara** [comment]
Yeah this should have made it into dev1. Just tested with the links in the About page, and it seems to work for me

**Comment by RDmitchell** [comment]
huh ... so clicking the links opens a new tab? when I do that, the program opens the link URL in the original tab. 

Maybe I need to restart my browser or something ??

**Comment by RDmitchell** [comment]
@adrian-lara -- The About page is good, but the Contacts page needs to be updated in the same way.

**Comment by aviveiros11** [comment]
Should be all set @adrian-lara and @RDmitchell 👍

**Comment by RDmitchell** [comment]
Thx !!

Let me know when this change is deployed to dev1 so I can do a final test
